tools/cvm-image-rewriter/plugins/06-install-tdx-guest-kernel/README.md

```diff
@@ -0,0 +1,26 @@
+# Install TDX guest kernel
+
+This plugin is used to install TDX guest kernel from a given local repository.
+
+- Prepare the local repository.
+
+Prepare the local repository and confirm that there are Debian packages related to the TDX kernel in the `/jammy/amd64/` directory of this repository.
+
+Then you can place this local repository in the `pre-stage/artifacts/` directory.(Optional) 
```

Comment:
I would remove Optional and have a clearer description up top of when this needs to run and not run and how to not run it.

Reply:
I saw further down you did explain how to not run.  Perhaps what is needed is a standard format for each plugin for how the readme is laid out. 

section 1 - What is the plugin
section 2 - How to enable or disable and whether it is optional
section 3 - How to configure the plugin and install